githooks/pre-push/helm-lint

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+currentBranch="$(git rev-parse --abbrev-ref HEAD)"
+originCommit="$(git rev-parse --short "$(git merge-base master "$currentBranch")")"
+filesToBePushed="$(git diff --name-only "$originCommit")"
+failed=false
+
+for chartName in $( cut -d'/' -f1,2 <<< "$filesToBePushed" | uniq ); do
+    # Avoid running 'helm lint|install' when modified dirs are not charts
+    if [[ $chartName = bitnami/* ]]; then
+        printf '\033[01;33mValidating %s:\n\033[0m' "$chartName"
+        chartPath="$(git rev-parse --show-toplevel)"/"$chartName"
+
+        printf '\033[0;34m- Running helm lint %s\n\033[0m' "$chartPath"
+        if helm lint "$chartPath"; then
+            printf '\033[0;32m\U2705 helm lint %s\n\n\033[0m' "$chartPath"
+        else
+            printf '\033[0;31m\U1F6AB helm lint %s failed. Pull Request cancelled.\n\n\033[0m' "$chartPath"
+            failed=true
+        fi
+
+        printf '\033\033[0;34m- Running helm install --dry-run %s\n\033[0m' "$chartPath"
+        helm repo add bitnami https://charts.bitnami.com/bitnami >> /dev/null
+        helm dependency update "$chartPath" >> /dev/null
+        if helm install --dry-run "$chartPath"; then
+            printf '\033[0;32m\U2705 helm install --dry-run %s\n\n\033[0m' "$chartPath"
+        else
+            printf '\033[0;31m\U1F6AB helm install --dry-run %s failed. Pull Request cancelled.\n\n\033[0m' "$chartPath"
+            failed=true
+        fi
+    fi
+done
+
+if $failed ; then
```

Comment:
You could make `failed` a numeric argument `0|1` and just do `exit $failed` at the end.